Docs 78277 Upated Kafka Example (#699)
mulesoft/docs-connectors: modules/ROOT/pages/kafka/kafka-connector-studio.adoc

include::_attributes.adoc[]
endif::[]

> In Anypoint Studio, add Kafka Connector to a Mule project, configure the connection to the Kafka cluster, and configure an input source for the connector.

Reviewer: Have we already used the full connector name?
(It never hurts to repeat it, however: up to you!)

|===
| Name | Type | Description | Default Value | Required
| Configuration | String | The name of the configuration to use. | | x

> | Consumption timeout a| Number |  The amount of time units that this operation will wait for receiving messages. |  |

Reviewer: ```suggestion
| Consumption timeout a| Number |  The number of time units that this operation waits to receive messages. |  |
```

** ALWAYS_JOIN
** JOIN_IF_POSSIBLE
** NOT_SUPPORTED |  The type of joining action that operations can take regarding transactions. |  `JOIN_IF_POSSIBLE` |
| Target Variable a| String |  The name of a variable to store the operation's output. |  |

> | Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable. |  `#[payload]` |

Reviewer: The description does not seem grammatically correct: an expression to evaluate...and store the expression outcome...? Please reevaluate.

|===
| Name | Description
| Connector Configuration |The name of the configuration to use.
| Poll timeout |The amount of time to block.
| Poll timeout time unit |  The time unit for the polling timeout. This combines with poll timeout to define the total timeout for the polling.
| Acknowledgment mode | Declares the supported acknowledgment mode type.
| Amount of parallel consumers | Declares how many consumers to use in parallel.
| Primary Node Only |Whether this source should be executed only on the primary node when running in a cluster.

> | Streaming Strategy a| Define the streaming strategy.

Reviewer: Imperative "define" or descriptive "defines"?
Make column entries parallel.
